### PR TITLE
feat: add fallback mechanism

### PR DESCRIPTION
**Pull request by thulieblack**
Add a fallback mechanism whereby if operationId is not available, we have the option to use the channel address. 

solves #2

### CODE REVIEW

**Comment by thulieblack** [comment]
@derberg I think I finally got it 😅😅😅

File: components/TopicFunction.js [hunk 1]
```diff
@@ -1,40 +1,62 @@
...
+*/
+
 export function TopicFunction({ channels }) {
+  const topicsDetails = getFunctionDetails(channels);
+  let functions = '';
+
+  topicsDetails.forEach(t => {
+    const functionName = t.functionName;
+
+    functions += `def ${functionName}(self, id):
```
Comment (derberg) [inline components/TopicFunction.js:14]:
```suggestion

    functions += `def ${t.functionName}(self, id):
```

**Comment by thulieblack** [comment]
I did the manual testing 
1. [Commit with only one operationID missing](https://github.com/derberg/python-mqtt-client-template/pull/11/commits/f2df780a0d62b6ae20f8546872415f1ee8749b1a)
2. [Commit with both operationID missing](https://github.com/derberg/python-mqtt-client-template/pull/11/commits/f0b3ec95168ab0b89af7e9cfb8d64f4e1cf2cb5d)

**Review by derberg — APPROVED** [review]
👏🏼

**Pull request merged**